frontend/src/features/leases/detail/LeasePages/insurance/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { Col } from 'react-bootstrap';
+import styled from 'styled-components';
+
+export const LabelCol = styled(Col)`
+  border-right: solid 1px ${props => props.theme.css.primaryColor};
+  padding-bottom: 10px;
```

Comment:
please convert all (except for media queries) px values to rem for consistency and accessibility support (1rem = 10px).